src/pybind/mgr/dashboard/frontend/src/app/ceph/block/iscsi/iscsi.component.spec.ts

```diff
@@ -38,7 +38,7 @@ describe('IscsiComponent', () => {
       FormatterService,
       RelativeDatePipe,
       IscsiBackstorePipe,
-      ListPipe,
+      JoinPipe,
```

Comment:
This was left behind somehow. It can be removed now.

Reply:
Done